src/bull_and_cows.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+/* eslint-disable no-console */
+
+const { generateNumber } = require('./randomNumber');
+
+const bullAndCows = (userNum) => {
+  const counters = {
+    cowCount: 0,
+    bullCount: 0,
+  };
+  const userNumArr = userNum.split('');
+  const randomNumArr = generateNumber.split('');
+
+  console.log(randomNumArr);
+
+  const tempArr = [...userNum];
+
+  randomNumArr.forEach((el, i) => {
+    if (el === tempArr[i]) {
+      counters.bullCount++;
+    }
+
+    if (userNumArr.includes(el)) {
+      counters.cowCount++;
+    }
+  });
+
+  return { ...counters };
```

Comment:
No need to recreate the object here.
```suggestion
  return counters;
```